changelog.md

```diff
@@ -110,6 +110,12 @@ with the `reviveOffers` Mesos scheduler API call. This should result in more spe
 for any reason you dislike this behavior, you can disable it with `--disable_revive_offers_for_new_apps`
 (not recommended).
 
+The `reviveOffers` and a `declineOffer` might be received by Mesos in the wrong order, so it should be repeated several
```

Comment:
`it should be repeated` is a bit ambiguous. I'd rather write something like
The order in which mesos receives `reviveOffers` and `declineOffer` calls is not guaranteed. Therefore, if we still need offers, the `reviveOffers` call will be repeated several times in order to lower the likelihood that mesos will receive a `declineOffer` afterwards (which was most probably sent first). You specify the number of repetitions with […]